### PR TITLE
R2s example add outputs

### PR DESCRIPTION
**Pull request by zxkjack123**
Update the R2S userguide according to the updated r2s example files: https://github.com/pyne/data/pull/17. 

- Geometry of the r2s example is changed
- Some folder names are changed

### CODE REVIEW

File: docs/usersguide/r2s.rst [hunk 1]
```diff
@@ -117,27 +117,21 @@ By using these sources for photon transport, the shutdown dose rate can be obtai
...
+There is a isotropic single energy (14 MeV) neutron point source in the center of the bottom right cube.
+The model is irradiated by a neutron source with intensitiy of 1e10 n/s for 3.5 days.
+The following content demonstrates the process of caculate the shutdown dose rate of this model at the time of 1 hour after shutdown.
 Example files could be found in 
-`r2s_example <https://github.com/pyne/data/blob/master/r2s_example.tar.gz>`_.
+`r2s_examples <https://github.com/pyne/data/blob/master/r2s_examples.tar.gz>`_.
 
 
-.. figure:: r2s_example_geometry.png
+.. figure:: r2s_example_geometry.pdf
```
Comment (gonuke) [inline docs/usersguide/r2s.rst]: I think this probably needs to be a `png` and not a `pdf` to render properly...

File: docs/usersguide/r2s.rst [hunk 2]
```diff
@@ -117,27 +117,21 @@ By using these sources for photon transport, the shutdown dose rate can be obtai
 PyNE R2S example
 ****************
 
-Using a simple geometry as a example, here is how we perform R2S calculation.
-The example geometry composed of four cubes of size 10x10x10 cm3. Cubes (in green) at bottom left and up right are water. Bottom right cube (in red) is made of steel, and up left cube (in white) is void.
-There is a isotropic single energy (14 MeV) neutron point source in the middle of the bottom right cube.
-The cubes are irradiated by the source with a single 3.5 day pulse, the neutron intensity is 1e10 n/s.
-The following content demonstrates the process of caculate the shutdown dose rate of these cubes at the time of 1 hour after shutdown.
+Using a simple geometry as a example, here is how we perform (voxel) R2S calculation.
+The example geometry composed of four cubes of size 10x10x10 cm3. It comsists of two different material, the water (in blue) and the steel (in red). The upper left cube (in white) is void.
```
Comment (gonuke) [inline docs/usersguide/r2s.rst]: consists
```suggestion
The example geometry is composed of four cubes of size 10x10x10 cm3. It consists of two different materials, the water (in blue) and the steel (in red). The upper left cube (in white) is void.
```

File: docs/usersguide/r2s.rst [hunk 3]
```diff
@@ -117,27 +117,21 @@ By using these sources for photon transport, the shutdown dose rate can be obtai
 PyNE R2S example
 ****************
 
-Using a simple geometry as a example, here is how we perform R2S calculation.
-The example geometry composed of four cubes of size 10x10x10 cm3. Cubes (in green) at bottom left and up right are water. Bottom right cube (in red) is made of steel, and up left cube (in white) is void.
-There is a isotropic single energy (14 MeV) neutron point source in the middle of the bottom right cube.
-The cubes are irradiated by the source with a single 3.5 day pulse, the neutron intensity is 1e10 n/s.
-The following content demonstrates the process of caculate the shutdown dose rate of these cubes at the time of 1 hour after shutdown.
+Using a simple geometry as a example, here is how we perform (voxel) R2S calculation.
```
Comment (gonuke) [inline docs/usersguide/r2s.rst]: should we introduce the concept of voxel vs. sub-voxel before describing "voxel" R2s?

File: docs/usersguide/r2s.rst [hunk 4]
```diff
@@ -117,27 +117,21 @@ By using these sources for photon transport, the shutdown dose rate can be obtai
...
+The example geometry composed of four cubes of size 10x10x10 cm3. It comsists of two different material, the water (in blue) and the steel (in red). The upper left cube (in white) is void.
+There is a isotropic single energy (14 MeV) neutron point source in the center of the bottom right cube.
```
Comment (gonuke) [inline docs/usersguide/r2s.rst]:
```suggestion
There is an isotropic monoenergetic (14 MeV) neutron point source in the center of the bottom right cube.
```

File: docs/usersguide/r2s.rst [hunk 5]
```diff
@@ -117,27 +117,21 @@ By using these sources for photon transport, the shutdown dose rate can be obtai
...
+The model is irradiated by a neutron source with intensitiy of 1e10 n/s for 3.5 days.
+The following content demonstrates the process of caculate the shutdown dose rate of this model at the time of 1 hour after shutdown.
```
Comment (gonuke) [inline docs/usersguide/r2s.rst]:
```suggestion
The following content demonstrates the process of caculating the shutdown dose rate of this model at the time of 1 hour after shutdown.
```

File: docs/usersguide/r2s.rst [hunk 6]
```diff
@@ -117,27 +117,21 @@ By using these sources for photon transport, the shutdown dose rate can be obtai
...
     **Figure 1:** *Geometry of R2S example (X-Y cross section).*
 
-Build the model using trelis as following figure. Set material group and export to `geom_without_material.h5m`.
-
-.. figure:: r2s_example_trelis_geometry.png
-    :align: center
-
-    **Figure 1:** *Trelis model for the example.*
-
+Build the model using trelis and set material group and export to `geom_without_material.h5m`.
```
Comment (gonuke) [inline docs/usersguide/r2s.rst]:
```suggestion
Build the model using Trelis (TM) and set material group and export to `geom_without_material.h5m`.
```
We may also want to add a link to Trelis and/or the DAGMC documentation

File: docs/usersguide/r2s.rst [hunk 7]
```diff
@@ -176,10 +170,11 @@ Perform R2S setup. The 'alara_params.txt' and 'config.ini' will be generated in
    $ r2s.py setup
 
 Modify important parameters in the 'alara_params.txt' and 'config.ini' according to the problem.
-Examples could be seen 'alara_params_example.txt' and 'config_example.ini' in 'r2s_example/r2s_run'.
+Examples could be seen 'alara_params_example.txt' and 'config_example.ini' in 'r2s_examples/r2s_run'.
```
Comment (gonuke) [inline docs/usersguide/r2s.rst]:
```suggestion
Examples can be seen in 'alara_params_example.txt' and 'config_example.ini' in 'r2s_examples/r2s_run'.
```

File: docs/usersguide/r2s.rst [hunk 8]
```diff
@@ -176,10 +170,11 @@ Perform R2S setup. The 'alara_params.txt' and 'config.ini' will be generated in
...
+By setting the 'sub_voxel' to be 'True', the user can perform sub-voxel R2S without any other change.
 Prepare alara nuclide library, copy preinstalled data library from
-`ALARA/data <`r2s_example <https://github.com/svalinn/ALARA/tree/master/data.>`_.
-Example nuclide library could be seen in 'r2s_example/r2s_run/data'
+`ALARA/data <`r2s_examples <https://github.com/svalinn/ALARA/tree/master/data.>`_.
```
Comment (gonuke) [inline docs/usersguide/r2s.rst]: this seems to have too many back ticks (`)?

**Review by gonuke — CHANGES_REQUESTED** [review]
A few language improvements.

You might want to update the tense from "could be" to "can be" throughout...

File: docs/usersguide/r2s.rst (hunk 8 above)
Comment (zxkjack123) [reply docs/usersguide/r2s.rst]: That's true. It has been updated now.

**Comment by zxkjack123** [comment]
@gonuke Made some changes to the sentences and the figures.

File: docs/usersguide/r2s.rst [hunk 9]
```diff
@@ -118,26 +118,19 @@ PyNE R2S example
 ****************
 
 Using a simple geometry as a example, here is how we perform R2S calculation.
-The example geometry composed of four cubes of size 10x10x10 cm3. Cubes (in green) at bottom left and up right are water. Bottom right cube (in red) is made of steel, and up left cube (in white) is void.
-There is a isotropic single energy (14 MeV) neutron point source in the middle of the bottom right cube.
-The cubes are irradiated by the source with a single 3.5 day pulse, the neutron intensity is 1e10 n/s.
-The following content demonstrates the process of caculate the shutdown dose rate of these cubes at the time of 1 hour after shutdown.
-Example files could be found in 
-`r2s_example <https://github.com/pyne/data/blob/master/r2s_example.tar.gz>`_.
+The example geometry is composed of four cubes of size 10x10x10 cm3. It consists of two different materials, the water (in blue) and the steel (in red). The upper left cube (in white) is void.
```
Comment (gonuke) [inline docs/usersguide/r2s.rst]:
```suggestion
The example geometry is composed of four cubes of size 10x10x10 cm\ :sup:`3`\ . It consists of two different materials, the water (in blue) and the steel (in red). The upper left cube (in white) is void.
```

**Review by gonuke — APPROVED** [review]

**Comment by gonuke** [comment]
Thanks @zxkjack123

**Pull request merged**